ksef_test.go

```diff
@@ -29,34 +29,34 @@ func TestNewDocument(t *testing.T) {
 		doc, err := test.NewDocumentFrom("invoice-pl-pl.json")
 		require.NoError(t, err)
 
-		bytes, bytes_err := doc.Bytes()
-		require.NoError(t, bytes_err)
+		bytes, bytesErr := doc.Bytes()
```

Comment:
In Go, the trend is to always use just `err` instead of defining the use-case with a prefix. As a rule, the error response is only ever used in the next few lines, so a short variable name is preferred.

In addition for this example, `bytes` is the name of a standard package in Go, so people tend to use `data` for `[]byte`.

Reply:
Good to know. I changed those names. It feels weird at first to redefine the same variable multiple times.

Reply:
😆 This is the way.